internal/pkg/api/server/templates/service/error.go

```diff
@@ -37,3 +42,19 @@ func (e BadRequestError) Error() string {
 func (e BadRequestError) ErrorUserMessage() string {
 	return e.Message
 }
+
+func NewValidationErrorFormatter() *utils.ErrorFormatter {
+	f := utils.NewErrorFormatter()
+	f.ErrorMessageFormatter(func(s string) string {
+		// Uppercase first letter
+		s = strhelper.FirstUpper(s)
+
+		// Add period if the message ends with an alphanumeric character
+		lastChar := s[len(s)-1:]
+		if regexpcache.MustCompile("^[a-zA-Z0-9]$").MatchString(lastChar) {
+			s += "."
+		}
+		return s
```

Comment:
Go chyby sa trocha naformatuju, prve pismeno je velke a veta konci bodkou.
Este bude potrebne neskor nejak vylepsit hlasky z validatora, ale to teraz neriesim.